src/jtag/drivers/bcm2835gpio.c

```diff
@@ -93,9 +97,9 @@ static void set_gpio_value(const struct adapter_gpio_config *gpio_config, int va
 	switch (gpio_config->drive) {
 	case ADAPTER_GPIO_DRIVE_MODE_PUSH_PULL:
 		if (value)
-			GPIO_SET = 1 << gpio_config->gpio_num;
+			GPIO_SET(gpio_config->gpio_num) = 1 << (gpio_config->gpio_num & 0x1f);
```

Comment:
IMHO it looks a bit weird seeing `gpio_config->gpio_num` repeated on both sides of the `=` sign? But perhaps "fixing" that would be too much of a departure from the way the existing code was working? :shrug:

Reply:
I agree - could rewrite entirely if you'd like and make set an entire macro?

Reply:
Not really my call to make. What does @P33M think?

Reply:
@P33M any thoughts?